autoload/fzf/vim.vim

```diff
@@ -1210,6 +1210,38 @@ function! fzf#vim#jumps(...)
   \ }, a:000)
 endfunction
 
+" ------------------------------------------------------------------
+" Changes
+" ------------------------------------------------------------------
+function! s:format_change(line)
+  return substitute(a:line, '\d\{1,2}', '\=s:yellow(submatch(0), "Number")', '')
+endfunction
+
+function! s:change_sink(lines)
+  if len(a:lines) < 2
+    return
+  endif
+  let cmd = s:action_for(a:lines[0])
+  if !empty(cmd)
+    execute 'silent' cmd
+  endif
+  let linecol = split(a:lines[1])
+  execute 'normal! '.linecol[1].'Gzz'
```

Comment:
* Doesn't work if the line starts with `>`: `>   0     1    8` ->  `0Gzz`
* We should also move the cursor to the designated column position. i.e. `call cursor(line, col)`
    * Note that we should not use `<COL>|` because it moves the cursor to the "screen column", not the real column number shown in `:changes`.